### DISCUSSION TITLE
[4.0] Indicating form controls as required using asterisks (*)

### DISCUSSION THREAD

**Discussion by pinak3748** [joomla/joomla-cms#35021] [2. Cold Features]
### What needs to be fixed
Indicating form controls as required using asterisks (*) should be red in color

### Why this should be fixed
A red asterisk means that filling in the field is mandatory in order to continue. If you don't fill in the required field, you will be asked to do it anyway. The field is automatically given a red border to indicate that you haven't filled it in yet. By seeing a red asterisk users instantly get that the field is required but if it's none colored it gets confusing in my opinion.

### How would you fix it
![after](https://user-images.githubusercontent.com/59458280/127806907-50ea5fe2-f7b5-4573-80d3-c658f9c07caf.jpg)

**Comment by brianteeman** [comment]
I disagree with the need for this change.

Also you are mistaken about when a field gets a red border. This happens whenever the field is invalid and is not related to it being required or not.

**Reply by pinak3748** [reply]
Ok, I got the field part wrong but I think the asterisks sign should be red, and sorry if I am getting it wrong.

**Comment by alikon** [comment]
i'll move it to discussion
